js/script.js

```diff
@@ -0,0 +1,55 @@
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+function renderBooks() {
+  const listDisplay = document.getElementById('list');
+  listDisplay.innerHTML = '';
+
+  books.forEach((book, index) => {
+    const bookDiv = document.createElement('div');
+    bookDiv.className = 'book';
+    const spanTitle = document.createElement('span');
+    spanTitle.textContent = book.title;
+    const spanAuthor = document.createElement('span');
+    spanAuthor.textContent = book.author;
+
+    const removeBtn = document.createElement('button');
+    removeBtn.className = 'btn';
+    removeBtn.textContent = 'Remove';
+    /* eslint-disable */
+    removeBtn.addEventListener('click', () => removeBook(index));
+    /* eslint-enable */
+    bookDiv.appendChild(spanTitle);
+    bookDiv.appendChild(spanAuthor);
+    bookDiv.appendChild(removeBtn);
+
+    listDisplay.appendChild(bookDiv);
+  });
+}
+
+function addBook() {
+  const titleInput = document.getElementById('title');
+  const authorInput = document.getElementById('author');
+  const title = titleInput.value;
+  const author = authorInput.value;
+
+  if (title && author) {
+    const bookData = {
+      title, author,
+    };
```

Comment:
- [OPTIONAL] I noticed you've added a new book to the collection, which is great. To enhance the management and future updates of our books, I recommend assigning a unique identifier, let's call it 'id,' to each new book. This identifier will prove incredibly helpful down the line when we need to update, search, or perform any unique operations on specific books. It ensures accuracy and avoids potential conflicts. So, let's implement this practice to make our book management system even more efficient and organized. Keep up the good work!

```suggestion
    const bookData = {
      title, author, id: Math.floor(Math.random() * 1000)
    };
```